### PR TITLE
test: refactor env tests to AVA

### PR DESCRIPTION
**Pull request by nfischer**
@freitagbr this is ready for review

### CODE REVIEW

File: ava-test/env.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+test.beforeEach(t => {
+  t.context.tmp = utils.getTempDir();
+  shell.config.silent = true;
+  shell.mkdir(t.context.tmp);
+});
+
+test.afterEach.always(t => {
+  shell.rm('-rf', t.context.tmp);
+});
```
Comment (freitagbr) [inline ava-test/env.js]: I don't think these tests need a temporary directory.

**Comment by nfischer** [comment]
@freitagbr ptal

**Comment by freitagbr** [comment]
LGTM

**Comment by nfischer** [comment]
LGTM

**Pull request merged**